pipelineutilities/pipelineutilities/standard_json_helpers.py

```diff
@@ -221,3 +222,25 @@ def _add_sequence(standard_json: dict) -> dict:
     if 'sequence' not in standard_json:
         standard_json['sequence'] = 0
     return standard_json
+
+
+def _insert_pdf_images(standard_json: dict):
+    tif_items = []
+    for item in standard_json.get('items', []):
+        if item.get('level') == 'file' and item.get('id', '').lower().endswith('.pdf'):
+            if 'sequence' not in item:
+                item['sequence'] = 1
+            item_clone = dict(item)
+            item_clone['id'] = item_clone.get('id').replace('.pdf', '.tif')
+            item['sequence'] = item.get('sequence') + 1
+            if 'thumbnail' in item:
+                item['thumbnail'] = False
+                item_clone['thumbnail'] = True
+            tif_items.append(item_clone)
+        if 'items' in item:
+            item = _insert_pdf_images(item)
+    if tif_items:
+        updated_file_path = standard_json.get('defaultFilePath', '').replace('.pdf', '.tif')
```

Comment:
It may be that a particular level of the tree doesn't have a defaultFilePath.  Because of this, I would recommend only updating if defaultFilePath exists.  So, maybe something like:

if tif_items and standard_json.get('defaultFilePath'):

instead of just:

if tif_items: